src/platform/platform_winuser.c

```diff
@@ -176,10 +176,11 @@ CxPlatProcessorInfoInit(
             if (Proc >= CxPlatProcessorGroupInfo[Group].Offset &&
                 Proc < CxPlatProcessorGroupInfo[Group].Offset + Info->Group.GroupInfo[Group].ActiveProcessorCount) {
                 CxPlatProcessorInfo[Proc].Group = Group;
-                CxPlatProcessorInfo[Proc].Index = (Proc - CxPlatProcessorGroupInfo[Group].Offset);
+                CXPLAT_DBG_ASSERT(Proc - CxPlatProcessorGroupInfo[Group].Offset < UINT8_MAX);
```

Comment:
I know its very unlikely to have a group with 256 threads, but shouldn't this check be <= UINT8_MAX?

Reply:
Good catch. I will update. Though, it is a debug assert, so it's SUPER unlikely to hit....